prawn-core.gemspec

```diff
@@ -0,0 +1,46 @@
+dir = File.expand_path(File.dirname(__FILE__))
+
+Gem::Specification.new do |spec|
+  spec.name = 'prawn-core'
+  spec.version = File.read(File.expand_path('VERSION', dir)).strip
+  spec.platform = Gem::Platform::RUBY
+  spec.summary = 'A fast and nimble PDF generator for Ruby'
+
+  spec.authors = [
+    'Gregory Brown',
+    'Guilherme Cassolato'
+  ]
+  spec.email = [
+    'gregory.t.brown@gmail.com',
+    'gcassolato@redhat.com',
+    '3scale-system-dev@redhat.com'
```

Comment:
Oh wait sorry 😖 , this is a private mailing list ... I dunno if we should do that.